services/PA-accounts-service/src/config.ts

```diff
@@ -4,7 +4,7 @@ export const AUTH_DEBUG = false;
 export const PORT = process.env.PORT_PA_ACCOUNTS_SERVICE;
 export const BASE_PATH = process.env.SUBDOMAIN_PA_ACCOUNTS_SERVICE + '/api';
 export const SCHEME =
```

Comment:
We should make the SCHEME itself configurable via an ENV-variable, so we don't need to have logic in our code for it.